dotNES/UI.cs

```diff
@@ -211,6 +229,10 @@ private void UI_MouseClick(object sender, MouseEventArgs e)
             {
                 MenuItems =
                 {
+                    new Item("Load Rom", x => x.Click += LoadRom)
```

Comment:
I think the name of this option should be "Load ROM...", perhaps even "&Load ROM..." to add a hotkey to it (referencing https://stackoverflow.com/a/637708, https://docs.microsoft.com/en-us/windows/desktop/uxguide/cmd-menus).